Noti/PushManager.swift

```diff
@@ -94,7 +94,7 @@ class PushManager: NSObject, WebSocketDelegate, NSUserNotificationCenterDelegate
             .responseString { response in
                 if let info = response.result.value {
                     debugPrint(info)
-                    self.userInfo = JSON.parse(info)
+                    self.userInfo = self.parseJSON(jsonString:info)
```

Comment:
Never call `.init` functions directly, use `JSON(parseJSON: info)` instead and remove parseJSON func please.